styles.css

```diff
@@ -1,179 +1,177 @@
-@import url('https://fonts.googleapis.com/css2?family=Ubuntu&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Ubuntu:wght@700&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Ubuntu:wght@400;700&display=swap");
 
-html, body {
+/* START GITHUB */
+.github-corner:hover .octo-arm {
+    animation: octocat-wave 560ms ease-in-out
+}
+
+@keyframes octocat-wave {
+
+    0%,
+    100% {
+        transform: rotate(0)
+    }
+
+    20%,
+    60% {
+        transform: rotate(-25deg)
+    }
+
+    40%,
+    80% {
+        transform: rotate(10deg)
+    }
+}
+
+@media (max-width:500px) {
+    .github-corner:hover .octo-arm {
+        animation: none
+    }
+
+    .github-corner .octo-arm {
+        animation: octocat-wave 560ms ease-in-out
+    }
+}
+
+/* END GITHUB */
+
+html,
+body {
     max-width: 100%;
     overflow-x: hidden;
 }
 
 body {
-	margin: 0;
+    font-family: "Ubuntu", sans-serif;
+    margin: 0;
+}
+
+.container {
+    max-width: 1024px;
```

Comment:
why max width?

Reply:
Because of containers, this way you dont have to add a margin to - for example - the lists, and have a consistant container vertical line. Imagine if you had a ultra-wide screen, this way it wont completely stretch